src/tcgwars/logic/impl/gen8/RebelClash.groovy

```diff
@@ -297,15 +297,9 @@ public enum RebelClash implements LogicCardInfo {
         move "Dark Guidance", {
           text "Choose a Basic Pokemon from your discard pile and play it onto your Bench."
           energyCost C
-          attackRequirement {
-            assert bench.notFull
-            assert my.discard.filterByType(BASIC)
-            }
+          attackRequirement {}
           onAttack {
-            def card = my.discard.findAll{it.basic}.select("Select the card to put on the bench").first()
-            my.discard.remove(card)
```

Comment:
why is dark guidance being reverted?

Reply:
My bad, was working with an outdated version.